plugins/catalog-backend/src/index.ts

```diff
@@ -27,3 +27,56 @@ export * from './processing';
 export * from './search';
 export * from './service';
 export * from './deprecated';
+
+import { LocationSpec as NonDeprecatedLocationSpec } from '@backstage/plugin-catalog-common';
+
+/**
+ * Holds the entity location information.
+ *
+ * @remarks
+ *
+ *  `presence` flag: when using repo importer plugin, location is being created before the component yaml file is merged to the main branch.
+ *  This flag is then set to indicate that the file can be not present.
+ *  default value: 'required'.
+ *
+ * @public
+ * @deprecated use the same type from `@backstage/plugin-catalog-common` instead
+ */
+export type LocationSpec = NonDeprecatedLocationSpec;
+
```

Comment:
were these lines really added as part of this PR? 🤔

Reply:
Good catch! I think it was a mistake during the rebase 👀 , it should be fixed now!